yroots/subdivision.py

```diff
@@ -129,6 +131,7 @@ def solve(funcs, a, b, rel_approx_tol=1.e-15, abs_approx_tol=1.e-12,
             target_deg = 3
 
     # Sets up the tolerances.
+    abs_approx_tol = max(abs_approx_tol,macheps)
```

Comment:
```suggestion
    if isinstance(abs_approx_tol, list):
        abs_approx_tol = [max(tol, 2*macheps) for tol in abs_approx_tol]
    else:
        abs_approx_tol = max(abs_approx_tol, 2*macheps)
```
Just because of the way that polishing is currently set up, having this makes it so that every abs_approx_tol passed into the function is no smaller than `2*macheps`, even when a list is passed in.